client/src/domain/Community/Community.js

```diff
@@ -0,0 +1,89 @@
+import "./Community.css";
+import { Link } from "react-router-dom";
+
+import CommunityFallbackProfilePic from "../../assets/happy-students.jpg";
+
+import { FaRegEdit } from "react-icons/fa";
+import { FaTrashAlt } from "react-icons/fa";
+import axios from "axios";
+
+import CommunitiesContext from "../../contexts/CommunitiesContext";
+import { useContext } from "react";
+
+import { useHistory } from "react-router";
+
+// TODO: Get rid of that conditional wrapper again - shoudl be possible to just set an onLick-handler on the section-tag itself and basrta kanzla!
+const ConditionalWrapper = ({ condition, wrapper, children }) =>
+  condition ? wrapper(children) : children;
+
+export default function Community({ community, as }) {
+  const renderAsListItem = as && as === "ListItem";
+  const communityProfilePicImgSrc = community?.picture
+    ? community?.picture
+    : CommunityFallbackProfilePic;
```

Comment:
Check out the [Nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to make this operation beautiful. ;-)

Reply:
Yeah - a simple logical or what have done the trick here as well - but that "Nullish coalescing operator" (what a beautiful beautiful name! Butterflies in my ears!) seems to be more error proof - so I go with that one - thanks for the tip, didn't know that one